test-utils/state-viewer/src/cli.rs

```diff
@@ -243,3 +247,40 @@ impl DumpAccountStorageCmd {
         );
     }
 }
+#[derive(Clap)]
+pub struct EpochInfoCmd {
+    /// Fetch the given epoch.
```

Comment:
I looked at `get_epoch_ids`, it looks like those options are exclusive.

Even if you specify `epoch_id`, `epoch_height` option will be ignored.
It look to me like, all those options should belong to a group and be mutually exlusive.